index.html

```diff
@@ -260,10 +268,48 @@ <h2>About <br />Myself</h2>
                 /></a>
               </li>
             </ul>
-            <button type="button">get my resume</button>
+            <button type="button">Get my resume</button>
           </div>
           <div class="tooling">
-            <img src="./skills.png" alt="my-skills" />
+            <div class="languages">
+              <ul>
+                <li class="about-list">
+                  <p>Languages</p>
+                  <i class="fas fa-chevron-down"></i>
+                </li>
+              </ul>
+            </div>
+
+            <div class="frames">
+              <img src="./img/Frame9(1).png" alt="" />

```

Comment:
- [ ] Please fix all the image `alt=" "`.

- Every image has alt with appropriate alternative text, that's useful for interacting or understanding the page has an alt text, for example, all the portfolio project images have an alt text.